patches/tools-typescript-ts_library.gni.patch

```diff
@@ -0,0 +1,17 @@
+diff --git a/tools/typescript/ts_library.gni b/tools/typescript/ts_library.gni
+index 90e8188a965d4199d4adc72327077efa7d64549b..266d24cd79d3a3dd05f28204a8997100cd76711a 100644
+--- a/tools/typescript/ts_library.gni
++++ b/tools/typescript/ts_library.gni
+@@ -116,8 +116,12 @@ template("ts_library") {
+     path_mappings = [
+       "chrome://resources/*|" +
+           rebase_path("$root_gen_dir/ui/webui/resources/tsc/*", target_gen_dir),
++      "chrome://resources/brave/*|" +
++          rebase_path("$root_gen_dir/brave/ui/webui/resources/tsc/*", target_gen_dir),
+       "//resources/*|" +
+           rebase_path("$root_gen_dir/ui/webui/resources/tsc/*", target_gen_dir),
++      "//resources/*|" +
++          rebase_path("$root_gen_dir/brave/ui/webui/resources/tsc/*", target_gen_dir),
```

Comment:
ditto
> Unfortunately the order of these items is important (I think), so I haven't combined to a single patch line at the end of the target definition
Although in this file we might be able to get away with it, purely by chance, because it seems to want the deeper paths underneath the parent paths, unlike `optimize_webui.gni`.